include/halo/lib/ir/tf_convert.td

```diff
@@ -248,4 +248,6 @@ def TF_Range : OpMapping<"Range", Range>;
 
 def TF_Sigmoid : OpMapping<"Sigmoid", Sigmoid>;
 
+def TF_Exp : OpMapping<"Exp", Exp>;
```

Comment:
Could you move it after "TF_Erf". We try tor keep the ops in alphabetic order. :)